src/dec128.rs

```diff
@@ -76,6 +78,46 @@ impl Hash for d128 {
     }
 }
 
+#[cfg(feature = "serde")]
+impl Serialize for d128{
+    fn serialize<S>(&self, serializer: &mut S) -> Result<(), S::Error>
+        where S: Serializer
+    {
+        serializer.serialize_str(&format!("{}", &self))
+    }
+}
+
+#[cfg(feature = "serde")]
+impl Deserialize for d128 {
+    fn deserialize<D>(deserializer: &mut D) -> Result<d128, D::Error>
+        where D: Deserializer
+    {
+        let s = try!(deserializer.deserialize_str(d128Visitor));
```

Comment:
Does this work? 

{ deserializer.deserialize_str(d128Visitor) work? }

(that is without try! or Ok(s))